packages/protect/package.json

```diff
@@ -53,7 +53,7 @@
 	},
 	"dependencies": {
 		"@byteslice/result": "^0.2.0",
-		"@cipherstash/protect-ffi": "0.16.0-0",
+		"@cipherstash/protect-ffi": "0.16.0-2",
```

Comment:
I'll kick off a release for v0.16.0 after the build finishes in protectjs-ffi main (not blocking for merging this one though).